app/src/main/java/com/alphawallet/app/ui/QRScanning/CameraPreview.java

```diff
@@ -309,4 +309,31 @@ public void onAutoFocus(boolean success, Camera camera) {
     private void scheduleAutoFocus() {
         mAutoFocusHandler.postDelayed(doAutoFocus, 1000);
     }
+
+    public void turnOffFlash() {
+        try {
```

Comment:
inconsistent bracing - the preferred code standard for AW is to use a separate line for bracing eg:
```
if (a=b)
{
    do something;
}
```